docker-compose.yml

```diff
@@ -27,6 +27,7 @@ services:
       - ./backend/core/.env
     image: postgres
     container_name: pgdb
+    shm_size: 1g
```

Comment:
Just curiosity really, but wondering if there was a problem that came up that caused you to need to increase the shm size?

Reply:
Initially, I was not using pagination and so it returned all the results from db. I received size limit error  as postgres container cound't store all the information and so i increased the size to 1GB. Some times the free text field is big and so even with pagination we may encounter huge size clinical records. That's why thought of increasing the size to 1GB for the worst case.